CMakeLists.txt

```
@@ -23,6 +23,15 @@ option(SENTRY_PIC "Build sentry (and dependent) libraries as position independen
 option(SENTRY_BUILD_TESTS "Build sentry-native tests" "${SENTRY_MAIN_PROJECT}")
 option(SENTRY_BUILD_EXAMPLES "Build sentry-native example(s)" "${SENTRY_MAIN_PROJECT}")
 
+if( LINUX )
```

```suggestion
if(LINUX)
```

Can you properly format this code block so that the whitespaces are consistent please?